src/please.go

```diff
@@ -339,6 +339,11 @@ var opts struct {
 				Targets []core.BuildLabel `positional-arg-name:"targets" description:"Targets to filter"`
 			} `positional-args:"true"`
 		} `command:"filter" description:"Filter the given set of targets according to some rules"`
+		Changed struct {
+			Since            string `long:"since" description:"Calculate changes since this tree-ish/scm ref (defaults to current HEAD/tip)."`
+			DiffSpec         string `long:"diffspec" description:"Calculate changes contained within given scm spec (commit range/sha/ref/etc)."`
+			IncludeDependees string `long:"include-dependees" default:"none" description:"Include direct or transitive dependees of changed targets."`
```

Comment:
FYI if this is an enum (which it looks like it is?) then you can do things like `choice:"none" choice:"direct"` to enforce it & get helpful help.